OneCommunityGlobal/HighestGoodNetworkApp
# Xiaoyu - Make moving summary to another week reset the summary checkmark

# Description

1. (PRIORITY MEDIUM) Jae: Make moving summary to another week reset the summary checkmark

a. If someone submitted their summary late for the week, and then forgot to submit it for the previous week and instead submitted it for this week, then it gets logged for this week and that (correctly) gives the green summary submitted checkmark. In cases like this, someone will sometimes need to move that summary to the last week. Right now, the green checkmark stays. If a summary is moved from this week to any other week, it should reset the checkmark. 

b. What we need is a "Move" button. This button will give the option to move the summary to "last week" or "week before last" or "choose date" and then open the submit modal with the information that is being moved.
Moving like this should clear the summary field for the current week and reset the green check so it’s clear the person still needs to submit a (new) summary for this week.

[Here's a video](https://www.dropbox.com/s/gu9m0mk9s2pqrdi/Move%20Summary%20Bug.mov?dl=0)

## Main changes explained:
- Add 'weekSelect' and creat radio including four options( 'this week', 'last week', Week Before Last', 'Three Weeks Ago'
- Create 'handleMoveOptions' function to reset the selected removed content and switch to weekSelect week.
- Disabled current week options in the current tab.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to dashboard→ 'Weekly Summary Due Date(click to open)→ select the rest three options, and check the content and green checkmark after submiting the summary.

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/bdba5509-7dd6-47af-b6fc-22e9e8ef05d1


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/7c4a487a-9d21-40e3-a8e0-d86b0664f105




## Note:
After submitting the summary, you may require to refresh the website to see the green checkmark change.
